Adyen.Test/UtilTest.cs

```diff
@@ -148,6 +128,56 @@ public void TestNullHmacValidator()
             Assert.IsFalse(isValidHmacAdditionalDataEmpty);
         }
         
+        [TestMethod]
+        public void TestColonAndBackslashHmacValidator()

```

Comment:
Good job. Enhancement (for bonus points) -> Keep changes like these in another branch; it makes reviewing individual (small) changes more practical for other engineers.